README.md

```diff
@@ -2,6 +2,18 @@
 
 A java integration library for AeroGear Digger
 
+## Project Info
+
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0  |
+| Build:          | Maven  |
+| Documentation:  | tbd  |
```

Comment:
I just meant that we doing client here, but missing reference for actual project information. Fully agree that we would need to build something on aerogear website. Maybe we can think about creating some backbone for now and then every PR and change would need involve documentation update? Just create some space that would be extended over the time? Do you think that this would be much trouble?